fleetctl/destroy.go

```diff
@@ -33,9 +33,14 @@ Destroyed units are impossible to start unless re-submitted.`,
 }
 
 func runDestroyUnits(args []string) (exit int) {
-	for _, v := range args {
-		name := unitNameMangle(v)
-		err := cAPI.DestroyUnit(name)
+	units, err := findUnits(args)
+	if err != nil {
+		stderr("%v", err)
+		return 1
+	}
+
+	for _, v := range units {
+		err := cAPI.DestroyUnit(v.Name)
```

Comment:
I thought that findUnits() is needed since it retrieves all units then it filters them, making sure to mangle names etc. It will only fail if it can't retrieve any unit all which is another problem. Or perhaps I'm missing something ?